mappings/net/minecraft/network/ClientConnection.mapping

```diff
@@ -94,6 +98,15 @@ CLASS net/minecraft/class_2535 net/minecraft/network/ClientConnection
 	METHOD method_36122 getOppositeSide ()Lnet/minecraft/class_2598;
 		COMMENT Returns the opposite side of this connection, or the direction of the packets
 		COMMENT sent by this connection.
+	METHOD method_36942 sendImmediately (Lnet/minecraft/class_2596;Lio/netty/util/concurrent/GenericFutureListener;Lnet/minecraft/class_2539;Lnet/minecraft/class_2539;)V
```

Comment:
```suggestion
	METHOD method_36942 sendInEventLoop (Lnet/minecraft/class_2596;Lio/netty/util/concurrent/GenericFutureListener;Lnet/minecraft/class_2539;Lnet/minecraft/class_2539;)V
```
Or `sendInternal` (similar to the style of our async render methods), as this is actually sending the packet after the code calling it has ensured it's only executed on the netty event loop